### PR TITLE
add lib/ to build; run qmake in cutechess/ cutechess/lib and cutechess/gui

### PR DESCRIPTION
**Pull request by tvanslyke**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,6 +1,9 @@
 edit:
-	cd stockfish && $(MAKE) build ARCH=$(ARCH)
-	cd cutechess/gui && $(MAKE)
+	cd stockfish && $(MAKE)
```
Comment (david-thornton) [inline Makefile]: Does this work? When I tried this it complained about not having the ARCH variable

Comment (david-thornton) [reply Makefile]: Yeah I just pulled down your branch and the Makefile doesn't work for me

Comment (tvanslyke) [reply Makefile]: Oops!  I must have accidentally removed the 'build ARCH=$(ARCH)'.  My latest commit fixes it

**Comment by david-thornton** [comment]
Looks good to me. Feel free to merge

File: Makefile [hunk 2]
```diff
@@ -1,6 +1,9 @@
 edit:
 	cd stockfish && $(MAKE) build ARCH=$(ARCH)
-	cd cutechess/gui && $(MAKE)
+	cd cutechess && qmake
+	cd cutechess/gui && qmake
+	cd cutechess/lib && qmake
+	cd cutechess && make
```
Comment (david-thornton) [inline Makefile]: Can you change this to $(MAKE)?

**Pull request merged**